src/hammer-vlsi/defaults.yml

```diff
@@ -48,6 +53,9 @@ vlsi.core:
   # LVS tool to use.
   lvs_tool: "nop"
 
+  # SRAM Compiler tool to use.
+  gen_srams_tool: "nop"
```

Comment:
How about `sramcompiler_tool`, `sramgenerator_tool` or `sram_gen_tool`? (we'll need to rename the above as well)